### PR TITLE
check isolated testing

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/sequential/crowdloan.API.test.ts [hunk 1]
```diff
@@ -52,23 +43,27 @@ beforeAll(async () => {
...
 
   testUser1.addAsset(nativeCurrencyId);
-  await Sudo.batchAsSudoFinalized(
-    Assets.mintNative(sudo),
-    Assets.mintNative(testUser1),
-    Assets.mintNative(testUser2),
-    Assets.mintNative(testUser3),
-    Assets.mintNative(testUser4),
-  );
+  // await Sudo.batchAsSudoFinalized(
```
Comment (goncer) [inline test/sequential/crowdloan.API.test.ts:61]: should this be still commented?

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/sequential/xyk-pallet.api.maintenance-config.test.ts [hunk 2]
```diff
@@ -266,53 +255,6 @@ test("maintenance- check we can sell MGX tokens and compoundRewards THEN switch
   );
 });
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply test/sequential/xyk-pallet.api.maintenance-config.test.ts:257]: Done

File: package.json [hunk 3]
```diff
@@ -133,7 +133,9 @@
     "test-microapps-prod-first": "export CHOPSTICK_UI=true ; rm nonce.db ; rm sudo.lock;  node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --verbose --runInBand --ci --group=microappsProdDeposit --group=microappsProdSwap --group=microappsProdWallet --group=microappsProdWithdraw --group=microappsProdStaking --group=microappsProdPools",
     "cliTool": "node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules cliTool/index.ts --runInBand",
     "test-rollup": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-seq; node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --verbose --runInBand --group=rollup --forceExit ",
+    "test-cookiesAndMilk": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-seq; rm nonce.db ; rm sudo.lock;  node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --verbose --runInBand --group=cookiesAndMilk --forceExit ",
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply package.json]: Done

File: test/sequential/crowdloan.API.test.ts [hunk 4]
```diff
@@ -52,23 +43,27 @@ beforeAll(async () => {
...
     await initApi();
   }
-  keyring = new Keyring({ type: "sr25519" });
+  keyring = new Keyring({ type: "ethereum" });
 
   api = getApi();
-  sudo = new User(keyring, sudoUserName);
+  sudo = new User(
+    keyring,
+    "0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133",
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply test/sequential/crowdloan.API.test.ts:51]: Done

File: test/sequential/crowdloan.API.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/sequential/crowdloan.API.test.ts:61]: I think, no, I've changed it

File: utils/globalSetup.js [hunk 5]
```diff
@@ -65,28 +65,28 @@ const globalConfig = async () => {
...
-        return Assets.registerAsset(`TEST_${i}`, `SYM_${i}`, 18);
-      }),
-  ];
-  const result = await Sudo.batchAsSudoFinalized(...txs);
-  testLog.getLog().info("Registered assets", result);
-  return result
-    .filter((X) => X.method === "RegisteredAsset")
-    .map((t) => t.eventData[0].data.toString());
-}
+// async function registerAssets(num = 300) {
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply utils/globalSetup.js]: It was temporary decision, uncommented

File: utils/setup.ts [hunk 6]
```diff
@@ -61,15 +61,16 @@ export function isBackendTest() {
...
     getEnvironmentRequiredVars().ethSudoAddress,
   );
 }
 export const setupUsers = () => {
-  keyring = new Keyring({ type: "sr25519" });
+  keyring = new Keyring({ type: "ethereum" });
   sudo = getSudoUser();
-  alice = new User(keyring, "//Alice");
-  eve = new User(keyring, "//Eve");
+  //alice = new User(keyring, "//Alice");
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply utils/setup.ts:71]: Done

File: utils/setup.ts [hunk 7]
```diff
@@ -78,8 +79,8 @@ export const setupUsers = () => {
   const testUser6 = new User(keyring);
 
   keyring.addPair(sudo.keyRingPair);
-  keyring.addPair(alice.keyRingPair);
-  keyring.addPair(eve.keyRingPair);
+  //keyring.addPair(alice.keyRingPair);
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply utils/setup.ts:82]: Done

File: utils/setup.ts [hunk 8]
```diff
@@ -184,10 +184,10 @@ export async function setupAPoolForUsers(users: User[]) {
   return { users, tokenIds };
 }
 export const setupGasLess = async (force = false) => {
-  keyring = new Keyring({ type: "ecdsa" });
-  const { sudo: sudoUserName } = getEnvironmentRequiredVars();
-  sudo = new User(keyring, sudoUserName) as EthUser;
-  alice = new User(keyring, "//Alice");
+  keyring = new Keyring({ type: "ethereum" });
+  sudo = getSudoUser();
+  //alice = new User(keyring, "//Alice");
```
[the remark this answers is not in the document]
Comment (AlexChetverov) [reply utils/setup.ts:189]: Done

**Review by goncer — APPROVED** [review]

**Pull request merged**